test/test_nestedtensor.py

```diff
@@ -3912,13 +3911,14 @@ def test_sdpa_compile(self, device, dtype):
         attn_d1 = torch.nn.functional.scaled_dot_product_attention(q_d1, k_d1, v_d1).transpose(1, 2)
         attn_d2 = torch.nn.functional.scaled_dot_product_attention(q_d2, k_d2, v_d2).transpose(1, 2)
 
-        compiled_sdpa = torch.compile(torch.nn.functional.scaled_dot_product_attention)
+        compiled_sdpa = torch.compile(torch.nn.functional.scaled_dot_product_attention, fullgraph=True)
         attn_nt = compiled_sdpa(q_nt, k_nt, v_nt).transpose(1, 2)
 
         attn_nts = attn_nt.unbind()
         self.assertEqual(attn_d1, attn_nts[0].unsqueeze(0), atol=output_ref_atol, rtol=output_ref_rtol)
         self.assertEqual(attn_d2, attn_nts[1].unsqueeze(0), atol=output_ref_atol, rtol=output_ref_rtol)
 
+    @xfailIfTorchDynamo
```

Comment:
You know what's up with this one?

Reply:
I'm not sure what is causing this yet, but I found a different issue that seems quite important to fix: 

`torch._dynamo.exc.Unsupported: Accessing overridden method/attribute values on a tensor subclass with a __torch_function__ override is not supported`

It makes everywhere we access `.values()` now graph break:
e.g., `output_dense = F.scaled_dot_product_attention(query.values(), key.values(), value.values())`

cc @mlazos